### ISSUE TITLE
on-cancel Callback

### ISSUE THREAD

**Issue by joelmuskwe** [troch/angular-multi-step-form#20]
Firstly great work awesome library. having challenges with on-cancel callback returning error TypeError: Cannot use 'in' operator to search for 'cancel()' in cancelled

I recently upgraded my angular to 1.4.8 and if i use 1.3.16 everything is working fine, please help

**Comment by troch** [comment]
What deos your custom on-cancel expression look like?

**Comment by joelmuskwe** [comment]
``` html
<multi-step-container steps="vm.directRechargeSteps" class="simple-prev-next" on-cancel="vm.clear()" on-finish="vm.complete()">
```

``` javascript
vm.clear = function(){
   console.log('cleared');
}
```

**Comment by troch** [comment]
Fixed with 1.2.7. Thanks for raising this.
